lib/swiss_schema.ex

```diff
@@ -448,11 +467,13 @@ defmodule SwissSchema do
         all.(__MODULE__, opts)
       end
 
+      @default_changeset Function.capture(__MODULE__, :changeset, 2)
```

Comment:
Does it make `__MODULE__` a compile dependency of SwissSchema? Because if it does, I believe it becomes a cyclic dependency issue with unexpected effects.

I'm not sure about it being an issue, but if it is, we can just stick with `Function.capture/3` directly in each function, just like before.